src/components/timeline/TrackScreenshots.js

```diff
@@ -300,6 +122,132 @@ class HoverPreview extends PureComponent<HoverPreviewProps> {
   }
 }
 
+type Props = {|
+  +threadIndex: ThreadIndex,
+  +windowId: string,
+|};
+
+export function TimelineTrackScreenshots({ threadIndex, windowId }: Props) {
```

Comment:
Should we use [React.memo](https://reactjs.org/docs/react-api.html#reactmemo) to check for the "own" prop changes? Maybe not useful for this specific component but it would be nice to build this habit.

Reply:
The more I read about hooks, the more I think we should use `React.memo` here :-)